frontend/src/metabase/search/components/SearchSidebar/DropdownSidebarFilter/DropdownSidebarFilter.tsx

```diff
@@ -134,15 +134,19 @@ export const DropdownSidebarFilter = ({
         ignoreTrigger
         autoWidth
         sizeToFit
+        pinInitialAttachment
+        horizontalAttachments={["right"]}
       >
-        <SearchEventSandbox>
-          <Box w={popoverWidth ?? "100%"}>
-            <ContentComponent
-              value={selectedValues}
-              onChange={selected => onApplyFilter(selected)}
-            />
-          </Box>
-        </SearchEventSandbox>
+        {popoverWidth && (
```

Comment:
Establishes a `minWidth` on the Box so we can have components that go outside of the 240px restriction if we need it. Also moved this to a `if` block since the `popoverWidth` will always be populated after the first load